### ISSUE TITLE
Crear tutorial configuración del proyecto en Windows

### ISSUE THREAD

**Issue by adrigm** [genbetadev/Genbeta-Dev-Engine#11]

**Comment by DavidBM** [comment]
Se incluiría la parte de añadir SFML?
Yo puedo hacer un tutorial de como dejar un MinGW con SFML listo para compilar con Sublime text editor 2. Con Makefile o sin makefile.

**Comment by adrigm** [comment]
Sí, se incluirá la parte de como añadir SFML. El proyecto usa SFML por lo que necesitamos saber integrar SFML en el proyecto que básicamente será incluir los archivos necesarios en las carpetas necesarias y usar una de las configuraciones que estén disponibles.

Harán falta tutoriales de como compilar SFML, si te animas puedes hacerlo.

**Comment by DavidBM** [comment]
Vale, puedo hacer un tutorial de como compilar SFML, tanto para linux como para windows con MinGW. 
Dime como estructuro las carpetaas y me pongo a ello.

**Comment by adrigm** [comment]
El tutorial es sobre como compilar la versión en desarrollo de SFML. Sería explicar como obtener el código fuente, usar cmake para contruir la solución y compilar las bibliotecas tanto estáticas como dinámicas.

**Comment by DavidBM** [comment]
Oki,
No se muy bien donde ubicarlo dentro de la jerarquia de carpetas del proyecto. Así que lo haré usando Markdown y cuando sepa donde ponerlo ya hago el pull.

**Comment by adrigm** [comment]
Ah vale no te entendía que querías decir. No, el tutorial no va dentro del proyecto. Va en la wiki: https://github.com/genbetadev/Genbeta-Dev-Engine/wiki

La tienes en el menú de la derecha aquí en Github, el icono del Libro abierto.

**Comment by DavidBM** [comment]
Hecho en falta la carpeta "lib" para poner los binarios de SFML. Habría que crearla, no?
Me pongo a hacer el tuturial de como bajarse las librerías de SFML, incluirlas y explicar un poco que es cada cosa.

**Comment by RdlP** [comment]
Los binarios son distintos según que Sistema Operativo uses, así que los binarios no deberían subirse a github

**Comment by adrigm** [comment]
@DavidBM van en extlibs/libs-xxx donde xxx es la plataforma en cuestión libs-vc2013, libs-mingw, ...

Los headers van en extlibs/headers/SFML

**Comment by DavidBM** [comment]
Ya tengo el tutorial de compilación de SFML en windows. Creo una página en la wiki llamada "Compilación de SFML en Windows" ?

**Comment by adrigm** [comment]
Sí, por favor. Gracias!

**Comment by DavidBM** [comment]
Ya está creada. Si ves algo que se puede mejorar, dimelo :)

**Comment by adrigm** [comment]
@DavidBM Está muy bien, quizás añadir como en vez de hacerlo desde la versión 2.1 que al fin y al cabo ya hay una descarga para obtener las librerías, podríamos hacerlo de la versión en desarrollo que suele ser mejor que la actual por el modo de trabajar del equipo de SFML. Es decir, el mismo tutorial, pero añadiendo al principio como obtener el código fuente del repositorio. Que en windows con la aplicación oficial de GitHub es bastante fácil.

**Comment by DavidBM** [comment]
Oki, pues lo modifico ahora y explico como bajarlo desde github. 
De hecho, opino que con el botón de donwload zip es incluso más facil ya que es algo que se compila muy de vez en cuando.

Además, ya me di cuenta hace tiempo que el github de sfml se actualizaba una vez al mes, es decir, no hay mucho movimiento.

**Comment by adrigm** [comment]
Al contrario, suele haber bastante movimiento. Por ejemplo, los commits de las últimas semana han sido muy importantes, se ha dado soporte a VC2013 y se han corregido varios bugs.

La versión zip nos da una instantánea del repositorio, si mañana arreglan un bug y cambian un archivo debemos bajarlo todo de nuevo. Si tenemos el repositorio sincroniado solo debemos bajar ese archivo, generar el proyecto y comilar.

**Comment by DavidBM** [comment]
Vaya, pues entonces me he tenido que confundir de proyecto.
Oki, pues ya lo he puesto. Explico como obtener el código de las 2 formas.

**Comment by angelnavarro** [comment]
Muchas gracias @DavidBM !! Ya tenemos un punto de partida en Windows ;-)
